### ISSUE TITLE
ubuntu 20.04 - device was not 0~3

### ISSUE THREAD

**Issue by leonsegal** [jergusg/k380-function-keys-conf#18]
the command which worked was: `sudo k380_conf -d /dev/hidraw4 -f on`

**Comment by jazzdup** [comment]
for me it was hidraw2, you likely just need to look in /dev first. This is great, thanks!

**Comment by leonsegal** [comment]
> for me it was hidraw2, you likely just need to look in /dev first. This is great, thanks!

Yes, it is usually hidraw{0~3} for me too, but on this one occasion it was hidraw4. Not sure how looking in /dev would suddenly make it hidraw2...unless l am missing something of course.
